controller/rancher/rancher_cert_dir_test.go

```diff
@@ -87,6 +87,22 @@ func TestCheckCertDirUpdate(t *testing.T) {
 		t.Fatalf("Error building config %v", err)
 	}
 
+	if len(newConfigs[0].Certs) != 3 {
```

Comment:
Only after both fullchain.pem and privkey.pem, the cert is included and the next check asserts that len(newConfigs[0].Certs) == 4.